### PR TITLE
docs: Add space-consumption-guideline

### PR DESCRIPTION
**Pull request by shuo-wu**
https://github.com/longhorn/longhorn/issues/6592

### CODE REVIEW

File: content/kb/space-consumption-guideline.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Besides, some operations like backup, rebuilding, or expansion, will lead to a hidden system snapshot creation. Hence there may be some snapshots even if users never create a snapshot for a volume manually.
+
+To eliminate space being wasted the historic data/snapshots, we would recommend applying a recurring job like `snapshot-delete` that limits the snapshot counts of volumes. You can check [the recurring job session](../../docs/1.5.1/snapshots-and-backups/scheduling-backups-and-snapshots) and see how to work.
+
+## Filesystem used size is much smaller than volume actual size
+
+The reason for this symptom is explained in [the volume size session](../../docs/1.5.1/volumes-and-nodes/volume-size/#volume-actual-size) as well. Briefly say, a volume is a block device, which does not recognize the filesystem. In other words, deleting a file from the filesystem does not mean that the related blocks can be released by the block device.
```
Comment (ejweber) [inline content/kb/space-consumption-guideline.md:24]: NIT: I had to read these sentences a couple of times to get the meaning.

```suggestion
The reason for this symptom is explained in [the volume size session](../../docs/1.5.1/volumes-and-nodes/volume-size/#volume-actual-size) as well. Briefly, a Longhorn volume is a block device which does not recognize the filesystem used on top of it. Deleting a file is a filesystem layer operation that does not actually free up blocks from the underlying volume.
```

**Review by ejweber — APPROVED** [review]
Very informative. I marked a couple of things that were a bit confusing to me on my initial readthrough. Please feel free to adopt my suggestions or ignore them.

File: content/kb/space-consumption-guideline.md (hunk 1 above)
Comment (shuo-wu) [reply content/kb/space-consumption-guideline.md:24]: Oh sorry for the confusion. Your statement is definitely easy to understand.

**Review by innobead — APPROVED** [review]
LGTM.

**Pull request merged**